copier/cli.py

```diff
@@ -75,6 +75,15 @@ class CopierApp(cli.Application):
             "the latest version, use `--vcs-ref=HEAD`."
         ),
     )
+    subdirectory = cli.SwitchAttr(
+        ["-D", "--subdirectory"],
```

Comment:
The `-D` option is already used by the `-D, --no-diff` option of the `update` subcommand, and I'm not sure both can live together. Any better idea than `D`? `s` and `d` are already taken. Other letters like `u`, `y` do not feel right. Maybe `b`? Or capital `S`?

Reply:
Both can coexist, but I prefer to choose a different one.

`b` is good IMHO.